Fixed sensitivity bug I found

conman180 opened the pull request:
Controls were locking up at certain speeds when they shouldn't
 Reduced max speed to 2.8 blocks/tick(problems with speeds in default worlds with chunks not loading fast enough).

conman180 committed:
Fixed sensitivity bug I found

Controls were locking up at certain speeds when they shouldn't did a couple workarounds.
 Reduced max speed to 2.8 blocks/tick(problems with speeds in default worlds with chunks not loading fast enough.

GoldSloth commented:
Thank you for fixing the bug you found, I'm happy with those changes.

For max speed, I don't think it should be limited, local servers can sometimes perform better chunk loading with lots of tweaking, and I think it's also up to content pack makers to set the speeds accordingly. I'd prefer to keep the limit the same, even if it borks in SP local worlds.

conman180 committed:
Remove Speed limiter

Removes speed limits. Up to pack makers to decide now. High speeds may cause issues.

GoldSloth commented:
Awesome, thank you :)
